lib/lhm/chunker.rb

```diff
@@ -16,7 +16,9 @@ class Chunker
     def initialize(migration, connection = nil, options = {})
       @migration = migration
       @connection = connection
-      @throttler = options[:throttler]
+      if @throttler = options[:throttler]
+        @throttler.connection = @connection if @throttler.respond_to?(:connection=)
```

Comment:
I dont think passing on initializer works, because you dont wanna touch the connection in a initializer. What you wanna do is use the same connection object as what the migrations are using.
So I would say this injection is the easier way to do it.